src/index.html

```diff
@@ -51,11 +58,13 @@
       text-align: center;
     }
     /* centers text on selection menu */
-    #selectIndustry h3, #selectFeature h3, #selectCompany h3 {
+    #selectIndustry h4, #selectFeature h4,
+    #selectCompany h4, #selectRegion h4 {
```

Comment:
h3 doesn't refer to anything anymore, must have been changed to h4 and CSS wasn't updated somewhere along the way